docs/reportData.html

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: Reports Data API
+---
+
+<p>Campaign provides an API to fetch reports data. Just like all APIs in the SDK, it's been wrapped into a function and will return a XML or JSON object based on the requested representation</p>
+
+<p>Function Definition</p>
+<pre class="code">
+client.getReport = (callContext: any, representation: string) : Promise<any>
+</pre>
+<p>callContext</p>
+<pre class="code">
+{
+  reportName: string<report Name>,
+  context: <"selection"|"global">,
+  selection: string<Delivery Id>,
```

Comment:
Reports are not just for deliveries. Documentation is misleading here. Also, I think selection can be more complex than just a single object: we support multiple objects for instance. Can you make sure this works for all possible selection types?